lib/net.c

```diff
@@ -67,9 +67,19 @@ close_num_ports(uint8_t nb_ports)
 	}
 }
 
+struct
+net_config *alloc_net_conf(void)
+{
+	struct net_config *net_conf = (struct net_config *)calloc(1, sizeof(*net_conf));
+	if (!net_conf)
```

Comment:
This `if` statement is doing no useful work. You can drop it, and the function will still do the exact same thing. So take advantage of it, drop the statement, and write in a single line:

`return calloc(1, sizeof(*net_conf));`

The only reason to avoid the single line is if we want to set default values other than zero.